services/marathon/marathon.go

```diff
@@ -39,6 +41,29 @@ type App struct {
 	Labels          map[string]string
 }
 
+const (
+	InternalHostnameLabel = "BAMBOO_INTERNAL_HOSTNAME"
+	ExternalHostnameLabel = "BAMBOO_EXTERNAL_HOSTNAME"
+)
+
+// Acl returns the internal, external, or service acl based on the
+// bamboo task configuration (whether the balancer is internal, or external),
+// or uses the configured acl for the service if neither of thoses cases is true.
+func Acl(haproxy conf.HAProxy, a App, service service.Service) string {
+	acl := service.Acl
+	internalHostname, hasInternal := a.Labels[InternalHostnameLabel]
```

Comment:
go question, is the second assignment here of `hasInternal` an `error` or `nil` type of thing?